script/reset-db.bash

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
```

Comment:
`set -euxo pipefail` is okay by me too, your call

Reply:
I prefer to write it out this way as I find it easier to read, I don't have to remember e, u, and x. In general I try to stick to long options for clarity.